deployments/add-ons/prometheus/README.md

```diff
@@ -0,0 +1,27 @@
+## Steps to setup Prometheus and Amazon Managed Service for Prometheus
+1. Create an IAM Policy:
+    1. `aws iam create-policy —policy-name AMPIngestPolicy —policy-document file://deployments/add-ons/prometheus/AMPIngestPermissionPolicy.json`
```

Comment:
In case customers might already have an AMPIngestPolicy, can we have the policy name as an environment variable for the setup? Also lets export the policy arn as a variable to pass in for the `eksctl create iamserviceaccount` command.

```
export AMP_POLICY_NAME=<desired-amp-policy-name>
export AMP_POLICY_ARN=$(aws iam create-policy --policy-name $AMP_POLICY_NAME --policy-document file://deployments/add-ons/prometheus/AMPIngestPermissionPolicy.json --query 'Policy.Arn' | tr -d '"')
```